requirements/test.txt

```diff
@@ -1,6 +1,3 @@
--r maintainer.txt
-coverage==3.7.1
-django-discover-runner
-mock
-sphinx-autobuild
+-r travis-ci.txt
+django
```

Comment:
This breaks the `tox` config. Can you remove it please. 

(I'll review unifying the travis and tox configs for 1.0)